### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zamith**

### CODE REVIEW

File: weeklyreport/README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 
-- [ ] Topics approached during the week.
-- [ ] Feedback from the mentor.
-- [ ] Feedback from the apprentice.
+* Topics covered during the week;
+* Feedback from the mentor;
+* Feedback from the apprentice.
 
-An online form is enough for the weekly report. The tool is optional, as long as it contains the necessary questions
+An online form is enough for the weekly report. The tool is optional, as long as it contains the necessary answers.
```
Comment (gabrielpoca) [inline weeklyreport/README.md:12]: I don't understand this change, it doesn't seem right.

Comment (zamith) [inline weeklyreport/README.md:12]: Maybe changing "contains" to "provides", but the reasoning is that we don't care about the questions, the output that we need are the answers.

**Comment by naps62** [comment]
@zamith @gabrielpoca bump

**Comment by naps62** [comment]
@gabrielpoca bump

**Comment by gabrielpoca** [comment]
I still don't agree with the change from answers to questions, but it doesn't seem like a problem :+1:

**Pull request merged**